CHANGELOG.md

```diff
@@ -4,8 +4,8 @@ All notable changes for major version updates will be documented here.
 
 ## 9.0.0
 
-- **Breaking:** function `registerBreadcrumbHelper` no longer takes any config used to define "default" breadcrumbs. All breadcrumbs are now sent to `res.render`.
-- **Breaking:** are changed, import with `const {registerBreadcrumbHelper} = require('@kth/kth-node-web-common/lib/handlebars/helpers/breadcrumbs')`
+- **Breaking:** function `registerBreadcrumbHelper` no longer takes any config. All breadcrumbs are now sent to `res.render`.
```

Comment:
Maybe highlight that is is sent to _every_ res.render

And that is something that has to be done manually.
(before, it was also sent to res.render, but automagically in the middleware)